src/todo/AddOrEditList.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useCallback } from "react";
+import { connect } from "react-redux";
+import * as actions from "../store/actions/list";
+import {
+  Typography,
+  TextField,
+  FormLabel,
+  Container,
+  InputLabel,
+  FormControl,
+  Button,
+  Box,
+  makeStyles,
+} from "@material-ui/core";
+
+import Lists from "./Lists";
+import AddOrEditTodoInList from "./AddOrEditTodoInList";
+
+const useStyles = makeStyles((theme) => ({
+  inputList: {
+    marginTop: theme.spacing(2),
+    minWidth: 300,
+  },
+  btnAddList: {
+    marginTop: theme.spacing(2),
+    marginLeft: theme.spacing(1),
+  },
+  root:{
+    display: "flex",
+    flexDirection:'column',
+    [theme.breakpoints.up("sm")]: {
+      flexDirection:'row',
+     },
+   }
+}));
+
+const AddOrEditList = (props) => {
+  const classes = useStyles();
+  const [id, setId] = useState(null);
+  const [name, setName] = useState("");
+  const callBackHandleEditList = useCallback(
+    (id) => {
+      console.log(id);
+      setId(id);
+      let lists = props.lists;
+      let data = lists.filter((item) => item._id === id);
+      setName(data[0].name);
+    },
+    [props.lists]
+  );
+
+  const handleEditList = () => {
+    console.log("Editing");
+    props.editList(id, name);
+  };
+  const handleAddList = () => {
+    console.log("Adding");
```

Comment:
avoid using console statement while pushing code to git repo

Reply:
done